src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryTool/Driver.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace BinaryTool;
+
+public partial class Driver
+{
+
+    // Target directory - where to look for binaries
+    private string TargetDirectory { get; set; }
+
+    // Allowed binaries keep file - binaries allowed in the target directory that should be kept
+    private string AllowedBinariesKeepFile { get; set; } = string.Empty;
+
+    // Allowed binaries remove file - binaries allowed in the target directory that should be removed
+    private string AllowedBinariesRemoveFile { get; set; } = string.Empty;
+
+    // Detected binaries file - list of detected binaries in the target directory
+    private string DetectedBinariesFile { get; set; }
```

Comment:
What's the thinking behind these being private properties? Usually for privates, plain readonly fields are used because there's no way to override them so having those with a dedicated setter/getter is useless.

This way it's also not clear whether these can be changed later (I believe they're not?).
Or if you want to keep properties for some reason, I'd remove the setter at least.